### ISSUE TITLE
Opening/Closing

### ISSUE THREAD

**Issue by simonrb2000** [benlamonica/homebridge-rasppi-gpio-garagedoor#27]
This is more of an enhancement, to open/close the garage door its a simple toggle relay. However your plugin/homekit/bridge is clever enough to know if its open and you say open the door it doesn't toggle. Same as asking it to close when close, it won't open it which is great. 

I was wondering when its closing (HomeKit says closing) could an open command be sent and it actually sends two toggle commands. The first stops it, then it toggles again to open it. So basically the plugin knows if its in an opening or closing state and a command is issued then it will send two toggle commands.. 

If its really complicated then I wouldn't worry, just a though I had lol

**Comment by benlamonica** [comment]
Yes, that could be done pretty easily. A simple check to see if it is currently closing or opening, and then triggering twice. I've been so swamped with life lately, but want to add new features soon. This is a good one and I'll add it.
